docs(admin): Configuration Parameter chapter refactoring (#12422)
nextcloud/documentation: admin_manual/configuration_server/config_sample_php_parameters.rst

   managed via individual apps.

Loading
-------

Configuration files located in ``config/`` are parsed automatically when Nextcloud 
starts up. They are also checked for changes periodically (approximately every two seconds 
in a standard PHP environment running with default *OPcache* settings; approximately every 
sixty seconds in many pre-packaged Nextcloud installation methods).

The ``config/config.php`` file may be supplemented by additional ``*.config.php`` files 
placed in the ``config/`` directory (if appropriately named and formatted).

.. danger:: Be cautious when naming or creating backup copies of your active 
   ``config/config.php``. If a backup is located within ``config/`` and is named
   ``(ANYTHING).config.php``, it will be loaded as part of your live configuration
   and override your ``config/config.php`` values!

.. tip:: If your configuration changes don't seem to be taking effect, check: (a) your PHP opcache 
   configuration; (b) for additional ``*.config.php`` files located in ``config/``; (c) the documentation
   for your Nextcloud installation method/package; (d) the output of ``occ config:list system``.

Format
------

> The short answer is that``config/`` files are plain text files with some special formatting 

Reviewer: A space is missing after `that` and `\`\`config/\`\`` (I hope you get it, I don't manage to escape this in the comment correctly)

Author: Got it. Thanks, @tflidd!

   from values being cast in unexpected ways.

Modifying
---------

Parameters may be modified in a standard text editor (i.e. via the command-line or externally 
then re-uploaded). They may also, in most cases, be modified using the commands in
the ``occ config:system:*`` namespace.

.. tip:: Incorrectly formatted ``key => value`` entries (or incorrectly specified values) may
   not generate immediate errors or problems (such as parsing / syntax errors), but may still 
   lead to unexpected and undesirable results. Review your fully parsed (by PHP) configuration
   by using the command ``occ config:list system`` and/or ``occ config:list system --private``
   to identify anything unexpected.

Defaults
--------

Nextcloud creates a base ``config/config.php`` file at installation time containing the most 
essential parameters for operations. These values are a mixture of auto-generated and drawn from
information provided by the administrator at installation time.

The file ``config/config.sample.php`` lists all the parameters within Nextcloud that can be 
specified in ``config/`` files, along with example and default values for each. The content of 

> that sample configuration file is included here for ease of reference and alongside additional 
> context.

Reviewer 2: Since it is some sections further, maybe add a link?
```suggestion
that sample configuration file is listed in the `sections below <#user-experience>`_ for ease of reference and
alongside additional context.
```

Author: Good call. Done. Added a stable id to use with a a `:ref:` to the config.php.sample section.
